prisma/migrations/20241207161306_init/migration.sql

```diff
@@ -4,9 +4,8 @@ CREATE TABLE "calls" (
     "remote_call_id" TEXT NOT NULL,
     "from" TEXT NOT NULL,
     "to" TEXT NOT NULL,
-    "started_at" TIMESTAMPTZ NOT NULL,
-    "ended_at" TIMESTAMPTZ,
-    "duration_in_ms" INTEGER,
+    "started_at" TIMESTAMP NOT NULL,
+    "ended_at" TIMESTAMP,
```

Comment:
Swapped `started_at` and `ended_at` to timestamp without timezone, so it gets stored as UTC and not dependant on the server's timezone. We can always add a timezone column to the call and extract the timezone from the phone number as a best guess.

DB was still empty so no harm 🤷‍♂️ .